business_register/serializers/company_serializers.py

```diff
@@ -89,7 +89,11 @@ def get_founder_of(self, company):
             return
         founded_companies = []
         for founder in founder_of:
-            founded_companies.append(founder.company.id)
+            founded_company_as_founder_of = self.get_founder_of(founder.company)
+            founded_companies.append({'id of the founded company': founder.company.id,
+                                      'name of the founded company': founder.company.name,
+                                      'companies founded by this company':
+                                          founded_company_as_founder_of})
```

Comment:
dont use spaces in dictionary keys

Reply:
change code style pls
```suggestion
            founded_companies.append({
                'id': founder.company.id,
                'name': founder.company.name,
                'founder_of': founded_company_as_founder_of
            })
```